frontend/src/pages/AddCardPage.js

```diff
@@ -33,24 +41,57 @@ export default function AddCardPage() {
     })
 
     return(
-    <form onSubmit={handleSubmit}>
-        <label>
-            Title:
-            <input type = "text" name = "title" value={title} onChange={e => setTitle(e.target.value)}/>
-        </label>
-        <label>
-            Description:
-            <input type= "text" name= "description" value={description} onChange={e => setDescription(e.target.value)}/>
-        </label>
-        <label>
-            Department:
-            <select name="selectList" id="selectList" value={department} onChange={e => setDepartment(e.target.value.toUpperCase())}>
-                <option value="--">--</option>
-                <option value="SALES">Sales</option>
-                <option value="HR">HR</option>
-            </select>
-        </label>
-        <button type="submit">Done</button>
-    </form>
+        <Container component="main" maxWidth="xs">
+            <CssBaseline />
+            <div >
+                <form className={classes.addCardForm} onSubmit={handleSubmit}>
+                        <TextField
+                            variant="outlined"
+                            margin="normal"
+                            required
+                            fullWidth
+                            label="Title"
+                            name="title"
+                            autoFocus
+                            value={title}
+                            onChange={e => setTitle(e.target.value)}
+                        />
+                    <TextField
+                            variant="outlined"
+                            margin="normal"
+                            required
+                            fullWidth
+                            multiline
+                            rows={5}
+                            label="Description"
+                            name="description"
+                            autoFocus
+                            value={description}
+                            onChange={e => setDescription(e.target.value)}
+                        />
+                    <FormControl className={classes.formControl}>
+                        <InputLabel shrink id="department select">
+                            Department
+                        </InputLabel>
+                        <Select
+                            name="department-select"
+                            labelId="department select"
+                            id="department-select"
+                            value={department}
+                            onChange={e => setDepartment(e.target.value.toUpperCase())}
```

Comment:
think you don't need `.toUpperCase()`since the values of MenuItem are already written in upper case

```suggestion
                            onChange={e => setDepartment(e.target.value)}
```